zipline/pipeline/loaders/base.py

```diff
@@ -17,5 +17,5 @@ class PipelineLoader(with_metaclass(ABCMeta)):
     TODO: DOCUMENT THIS MORE!
     """
     @abstractmethod
-    def load_adjusted_array(self, columns, mask):
+    def load_adjusted_array(self, columns, dates, assets, mask):
```

Comment:
Good catch here.  Some day I want to write a better `ABCMeta` that cares about method signature compatibility.